language-support/java/codegen/BUILD.bazel

```diff
@@ -330,21 +328,6 @@ java_test(
     ],
 )
 
-java_test(
-    name = "integration-tests-contract-keys",
-    srcs = [contract_keys_test],
-    test_class = "com.daml.ContractKeysTest",
-    deps = [
-        ":integration-tests-model-2.dev",
-        "//canton:bindings-java",
-        "@maven//:com_google_protobuf_protobuf_java",
-        "@maven//:org_junit_jupiter_junit_jupiter_api",
-        "@maven//:org_junit_jupiter_junit_jupiter_engine",
-        "@maven//:org_junit_platform_junit_platform_commons",
-        "@maven//:org_junit_platform_junit_platform_runner",
-    ],
-)
```

Comment:
no need to create a special case for contract key, just need to make it a 2.dev test.

Reply:
Oh right! I did this change mechanically without looking at the larger picture.